spec/models/channel_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Channel, type: :model do
+  include Rails.application.routes.url_helpers
+
+  before { allow(GoogleOauth).to receive(:push_notification_client) }
+
+  subject(:channel) { create :channel }
+  describe 'factory' do
+    it 'creates a valid channel' do
+      expect(channel).to be_valid
+    end
+  end
+
+  describe 'validation' do
+    %i(channel_id resource_id expiration conference_room_id).each do |s|
```

Comment:
I'm not a big fan of iterating in specs buut these are that simple that I can suck it up.